src/mqtt_bridge/bridge.py

```diff
@@ -103,9 +103,14 @@ def _register_service(self, client, userdata, mqtt_msg):
 
         if msg_dict['op'] == 'register':
             rospy.loginfo("register service proxy")
-            self._bridges.add(RemoteService(
-                **msg_dict['args'])
-            )
+            try:
+                self._bridges.add(RemoteService(
+                    **msg_dict['args'])
+                )
+            except rospy.ServiceException as e:
+                rospy.logerr("Captured exception when trying to register a "
+                "service twice. This happens when mqtt clients are restarted:"
+                 " %s" % (e,))
```

Comment:
See above. The "correct" way should be to deregister Services if and when a disconnection happens. Then also this wouldn't be a problem.